src/applications/resources-and-support/tests/components/ResourcesAndSupportSearchApp.unit.spec.jsx

```diff
@@ -35,59 +31,5 @@ describe('ResourcesAndSupportSearchApp', () => {
   it('creates a landmark for the search form', async () => {
     const screen = renderInReduxProvider(<ResourcesAndSupportSearchApp />);
     await screen.findByLabelText('Enter a keyword, phrase, or question');
-
-    screen.getByRole('search');
-  });
```

Comment:
Removed because now the search input is in the shadow DOM and we can't test much in unit tests.